<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HCHARTVIEW</class>
 <widget class="QWidget" name="HCHARTVIEW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>786</width>
        <height>643</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="filtresVal">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>350</height>
          </size>
         </property>
         <property name="title">
          <string>Valeurs des filtres</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QGroupBox" name="sexeB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Sexe</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="sexe">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>Femme</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Homme</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="ageB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>25</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Age</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="age">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="age2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QGroupBox" name="villeB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Ville</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="ville">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="quartierB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="maximumSize">
                    <size>
                     <width>42</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Quartier</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="quartier">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QGroupBox" name="nationaliteB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="maximumSize">
                    <size>
                     <width>55</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Nationalite</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="nationalite"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="professionB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="maximumSize">
                    <size>
                     <width>55</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Profession</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="profession">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QGroupBox" name="dateAdmissionB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="maximumSize">
                    <size>
                     <width>85</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Date d'admission</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="dateAdmission">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="dateAdmission2">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QGroupBox" name="modeSoinB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Mode de Soin</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="modeSoin"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="modeSortieB">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_17">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Mode de sortie</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="modeSortie"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QGroupBox" name="queries">
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QGroupBox" name="infoMedB">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Information Medicale</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="queriesContainerLayout">
                      <item>
                       <widget class="QComboBox" name="queryBase">
                        <property name="maximumSize">
                         <size>
                          <width>200</width>
                          <height>20</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Valeurs</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="values">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="editable">
                       <bool>true</bool>
                      </property>
                      <property name="insertPolicy">
                       <enum>QComboBox::NoInsert</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QToolButton" name="addQuery">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset resource="../../hom_ressources.qrc">
                        <normaloff>:/images/plus</normaloff>:/images/plus</iconset>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QToolButton" name="delQuery">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                      <property name="icon">
                       <iconset resource="../../hom_ressources.qrc">
                        <normaloff>:/images/moins</normaloff>:/images/moins</iconset>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="lauch">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>calculer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Paramétrer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="paramFiltre">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic Light</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Paramètre de filtre pour l'étude statistique</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="0">
           <widget class="QCheckBox" name="professionF">
            <property name="text">
             <string>Profession</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="villeQuartierF">
            <property name="text">
             <string>Ville et Quatier</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="IntervalleAdmissionF">
            <property name="text">
             <string>Intervalle de date d'admission</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="dateAdmissionF">
            <property name="text">
             <string>Date d'admission</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="ageF">
            <property name="text">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="sexeF">
            <property name="text">
             <string>Sexe</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="nationaliteF">
            <property name="text">
             <string>Nationalité</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="message">
            <property name="text">
             <string>Selectionnez en premier le caractéristique principale de l'étude</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="intervalleAgeF">
            <property name="text">
             <string>Intervalle d'âge</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="villeF">
            <property name="text">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="rechoisir">
            <property name="text">
             <string>Rechoisir le cractéristique principale</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="refresh">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="endPram">
            <property name="text">
             <string>Finir</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="modeSoinF">
            <property name="text">
             <string>Mode de soin</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="modeSortieF">
            <property name="text">
             <string>Mode de sortie</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QCheckBox" name="informationMedicalF">
            <property name="text">
             <string>Infomation médicale</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="containerChart"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../hom_ressources.qrc"/>
 </resources>
 <connections/>
</ui>
